src/ComponentSidebar.js

```diff
@@ -119,8 +119,8 @@ class ComponentSidebar extends Component {
       content: 'FAB',
       url: '/fab',
     }, {
-      content: 'Icon Toggle',
-      url: '/icon-toggle',
+      content: 'Icon Button',
```

Comment:
This was updated, but `ComponentImageList` needs to be updated as well - it still says Icon Button